src/main/java/nitezh/ministock/dataaccess/RssItem.java

```diff
@@ -0,0 +1,50 @@
+package nitezh.ministock.dataaccess;
+
+/**
+ * Created by nicholasfong on 2018-04-04.
+ */
+
+public class RssItem {
+    private String title;
+    private String description;
+    private String link;
+
+    public RssItem(){
```

Comment:
unnecessary duplication of constructor.
do `this("","","");` for RssItem() constructor rather than initialising each variable as empty . will make it easier to modify later on